CHANGES/3629.feature

```diff
@@ -1 +1 @@
-Add support for Named Pipes(Site and Connector) on windows.
+Add support for Named Pipes (Site and Connector) under Windows. This feature requires Proactor event loop to work.
```

Comment:
Do you know you can just hit "accept patch suggestion" on GitHub UI and not have to copy-paste everything manually?

Reply:
No I didn't!
Thanks for telling me :-D